components/bl-weather-component/styles/index.less

```diff
@@ -0,0 +1,9 @@
+@bl-customComponent-weather-size: 100%;
+
+.bl-customComponent-weather {
+  width: @bl-customComponent-weather-size;
+
+  .rw-container {
+    width: @bl-customComponent-weather-size;
```

Comment:
If the user has set @bl-customComponent-weather-size to 50%, we will get 50% from bl-customComponent-weather, not from the component container.